Kros.Utils/tests/Kros.Utils.UnitTests/Data/BulkInsert/SqlServerBulkHelper.cs

```diff
@@ -61,20 +75,35 @@ private static void CompareColumns(DataTable actualTable, DataTable expectedTabl
                 $"expected = {expectedTable.Rows.Count}.");
 
         private static void CompareData(DataTable actualTable, DataTable expectedTable)
+        {
+            foreach (DataRow expectedRow in expectedTable.Rows)
+            {
+                CompareRowOneColumn(actualTable, expectedTable, expectedRow);
+            }
+        }
+
+        public static void CompareRowOneColumn(DataTable actualTable,
+                                               DataTable expectedTable,
+                                               DataRow expectedRow)
+        {
+            CompareRowOneColumn(actualTable, expectedTable, expectedRow, string.Empty);
+        }
+
+        public static void CompareRowOneColumn(DataTable actualTable,
```

Comment:
Do not use Visual Basic style parameters breaks. If the will not fit into one line do it [this way](https://github.com/Kros-sk/Kros.Libs/blob/master/Kros.Utils/src/Kros.Utils/Data/Schema/ForeignKeySchema.cs#L37-L43).